### PR TITLE
file organization

### PR DESCRIPTION
**Pull request by Rhegan**
No change except file structure and adding a title and prep to add logo to titlebar

### CODE REVIEW

**Comment by DominicPrzygonski** [comment]
What are all the files in the idea folder required for?

**Comment by DominicPrzygonski** [comment]
And does the project still run and load fine after all the file changes?

**Comment by Rhegan** [comment]
I think those are files from the ide that weren't added to the gitIgnore. As far as i have experienced it is all the same and runs correctly since it's just the routing that has chaged. I have noticed though that the dog images are not showing, but i am busy fixing that now. It's just routing. Let me fix it and ill do a second PR

**Comment by DominicPrzygonski** [comment]
We shouldn't really need a gitingore as html is just run in the browser so we dont really need to exclude any files. An idea shouldn't really add anything to the project as its all static html at the moment

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Default ignored files
+/shelf/
+/workspace.xml
```
Comment (DominicPrzygonski) [inline .idea/.gitignore]: We dont need any of the files in the .idea folder at the moment you can include them in a separate commit if the are needed later

**Pull request merged**